### ISSUE TITLE
Global variables are not assigned correctly in interpreter.

### ISSUE THREAD

**Issue by joshuacrotts** [JoshuaCrotts/LittleC-Compiler#6]
Example:

Assume x, y, and z are global variables initializing to 0, and we perform the following statement in main():
x = 5 
results in x still being 0.

**Comment by joshuacrotts** [comment]
Correction - it does not work when the assignee is declared _below_ another global variable. For example:

```
int k;
int i;

void main() {
	k = 5;
	printd(k);
}
```

Prints 5 correctly. However,
```
int i = 3;
int k;

void main() {
	k = 5;
	printd(k);
}
```

erroneously prints 3! It seems to be a problem with how the offsets are handled with global pointers.

**Comment by joshuacrotts** [comment]
Fixed bug in 46c20b1.
